xcode/Subconscious/Shared/Components/Common/Byline/PetnameView.swift

```diff
@@ -10,27 +10,20 @@ import SwiftUI
 /// Byline style for displaying a petname
 struct PetnameView: View {
     var petname: Petname
+    var address: Slashlink?
     
     var body: some View {
-        let parts = petname.parts()
-
-        HStack(alignment: .lastTextBaseline, spacing: 0) {
-            let first = parts[0]
-            
-            Text(first.markup)
-                // Fixed size to ensure truncation trims path preferentially
-                .fixedSize(horizontal: true, vertical: false)
-                .lineLimit(1)
-            
-            let rest = parts[1...]
-                .map { p in p.description }
-                .joined(separator: ".")
-            
-            if rest.count > 0 {
-                // Particular structure to ensure truncation trims the path and never the name
-                Text(".\(rest)")
-                    .lineLimit(1)
```

Comment:
We should come back and re-think this logic as part of https://github.com/subconsciousnetwork/subconscious/issues/615